tests/docker_wrapper.py

```diff
@@ -268,3 +275,57 @@ def redis_docker(mounts=None):
     redis_wrap.test_redis_cnx()
 
     return redis_wrap
+
+
+class RabbitMQDockerWrapper(DockerWrapper):
+    def __init__(
+        self,
+        image_name,  # type: unicode
+        container_name=None,  # type: Optional[unicode]
+        dockerfile_obj=None,  # type: Optional[IO]
+        dockerfile_path=None,  # type: Optional[unicode]
+        env_vars={},  # type: Dict[unicode, unicode]
+        mounts=None,  # type: Optional[List[docker.types.Mount]]
+    ):
+        # type: (...) -> None
+        super(RabbitMQDockerWrapper, self).__init__(
+            image_name,
+            container_name,
+            dockerfile_obj,
+            dockerfile_path,
+            env_vars,
+            mounts,
+            ports={"5672/tcp": ("0.0.0.0", "5672")},
+        )
+        protocol = "pyamqp"
+        username = "guest"
+        password = "guest"
+        port = 5672
+        url = "{0}://{1}:{2}@{3}:{4}//".format(protocol, username, password, self.ip_addr, port)
```

Comment:
minor: I would add (to stick to usual config)
```suggestion
        url = "{0}://{1}:{2}@{3}:{4}//?heartbeat=60".format(protocol, username, password, self.ip_addr, port)
```

Reply:
Done in e44b26456cc1e1214e4cfbca9aeb78b04a484245.